rocky-backend/DBtest.py

```
@@ -0,0 +1,95 @@
+    test("Update user", res.status_code == 200)
+
+
+# update user bad data
+if user_id:
+    res = requests.put(f"{BASE}/users/{user_id}", json={"role": 123})
+    test("Reject bad update", res.status_code == 400)
+
+
+# delete user
+if user_id:
+    res = requests.delete(f"{BASE}/users/{user_id}")
```

Ruff (0.15.7)

[error] Probable use of `requests` call without timeout

(S113)